provider/ec2/environ.go

```diff
@@ -2527,31 +2460,30 @@ func (e *environ) ensureGroup(ctx context.ProviderCallContext, controllerUUID, n
 		vpcIDParam = aws.String(chosenVPCID)
 	}
 
+	// Tag the created group with the model and controller UUIDs.
+	cfg := e.Config()
+	tags := tags.ResourceTags(
+		names.NewModelTag(cfg.UUID()),
+		names.NewControllerTag(controllerUUID),
+		cfg,
+	)
```

Comment:
This is used a few places, should be a helper to get the base tags from a model config

Reply:
Not sure it will make a difference as we still need the config. This is just a copy and paste from what we had before.